src/commands/new.ts

```diff
@@ -152,8 +162,6 @@ export default {
     // #endregion
 
     // #region Project Name
-    heading("🔥 About to Ignite your new app! 🔥")
```

Comment:
Could we add two blank lines before the initial question? It feels weird being smashed up against the command prompt.

```ts
p()
p()
```

Reply:
Yeah I can add that!

Reply:
I like visual space -- as you can tell :D

Reply:
<img width="464" alt="Screen Shot 2022-09-09 at 10 32 40 AM" src="https://user-images.githubusercontent.com/37849890/189410657-100f8d5c-07d1-4ccf-a7b5-d47bcf06eef4.png">

[feat(new): add a line break before question prompts](https://github.com/infinitered/ignite/pull/2119/commits/15a89a7c0ea541b4a66eee6d30f2d01c05497036)